src/renderer/fonts/font_options.rs

```diff
@@ -40,10 +43,13 @@ impl FontOptions {
                 bold = true;
             } else if part == "i" {
                 italic = true;
+            } else if part == "o" {
```

Comment:
i would replace this with `use-italic-as-oblique` or `i-as-o`, becouse simple `o` could be interpretered as oblique.

Reply:
Makes sense to me